### PR TITLE
⌗118598 Accept `edit_event_tickets` Permission in Metabox Permissions Check

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**⌗118598**](http://central.tri.be/issues/118598)

### CODE REVIEW

File: src/Tribe/Metabox.php [hunk 1]
```diff
@@ -474,7 +475,13 @@ public function has_permission( $post, $data, $nonce_action ) {
 			$post = get_post( $post );
 		}
 
-		return ! empty( $data['nonce'] ) && wp_verify_nonce( $data['nonce'], $nonce_action ) && current_user_can( get_post_type_object( $post->post_type )->cap->edit_posts );
+		if ( empty( $data['nonce'] ) ) {
```
Comment (mitogh) [inline src/Tribe/Metabox.php]: I think you can do a small refactor here to make it a bit more compact

```suggestion
		if ( empty( $data['nonce'] ) || ! wp_verify_nonce( $data['action'], $nonce_action ) ) {
			return false;
		}
```

File: src/Tribe/Metabox.php [hunk 2]
```diff
@@ -474,7 +475,13 @@ public function has_permission( $post, $data, $nonce_action ) {
...
+			return false;
+		}
+
+		$has_permission = current_user_can( 'edit_event_tickets' ) || current_user_can( get_post_type_object( $post->post_type )->cap->edit_posts );
```
Comment (mitogh) [inline src/Tribe/Metabox.php]: So this one can be converted into:

```suggestion
		return current_user_can( 'edit_event_tickets' ) || current_user_can( get_post_type_object( $post->post_type )->cap->edit_posts );
```

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *Sprint 5 (R2 Release)*, but this was pull requested against *release/M18.17*.

**Comment by mitogh** [comment]
Changed the base of this PR as main ticket is part of R5 release.

**Comment by mitogh** [comment]
Changed the base of this PR @ggwicz let me know if you can address those points otherwise I will take care.

Thanks.

**Pull request closed without merging**